plugins/saveactions.c

```diff
@@ -718,53 +1373,116 @@ GtkWidget *plugin_configure(GtkDialog *dialog)
+		image = gtk_image_new_from_stock(GTK_STOCK_OPEN, GTK_ICON_SIZE_BUTTON);
+		gtk_container_add(GTK_CONTAINER(button), image);
+
+		hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 6);
+		gtk_box_pack_start(GTK_BOX(hbox), entry_dir, TRUE, TRUE, 0);
+		gtk_box_pack_start(GTK_BOX(hbox), button, FALSE, FALSE, 0);
+
+		gtk_box_pack_start(GTK_BOX(inner_vbox), hbox, FALSE, FALSE, 0);
+
+		label = gtk_label_new_with_mnemonic(
+			_("Date/_Time format for backup files (for a list of available conversion specifiers see https://docs.gtk.org/glib/method.DateTime.format.html):"));
+		gtk_misc_set_alignment(GTK_MISC(label), 0, 0.5);
+		gtk_box_pack_start(GTK_BOX(inner_vbox), label, FALSE, FALSE, 7);
+
+		pref_widgets.backupcopy_entry_time = entry_time = gtk_entry_new();
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), entry_time);
+		if (!EMPTY(backupcopy_time_fmt))
+			gtk_entry_set_text(GTK_ENTRY(entry_time), backupcopy_time_fmt);
+		gtk_box_pack_start(GTK_BOX(inner_vbox), entry_time, FALSE, FALSE, 0);
+
+		hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 6);
+
+		label = gtk_label_new_with_mnemonic(
+			_("Directory _levels to include in the backup destination:"));
+		gtk_misc_set_alignment(GTK_MISC(label), 0, 0.5);
+		gtk_box_pack_start(GTK_BOX(hbox), label, FALSE, FALSE, 0);
+
+		spin_dir_levels = gtk_spin_button_new_with_range(0, 20, 1);
+		pref_widgets.backupcopy_spin_dir_levels = spin_dir_levels;
+		gtk_spin_button_set_value(GTK_SPIN_BUTTON(spin_dir_levels), backupcopy_dir_levels);
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), spin_dir_levels);
+		gtk_box_pack_start(GTK_BOX(hbox), spin_dir_levels, FALSE, FALSE, 0);
+
+		gtk_box_pack_start(GTK_BOX(inner_vbox), hbox, FALSE, FALSE, 7);
+	}
+	/*
+	 * Untitled Document Save
+	 */
+	{
+		GtkWidget *disabled_radio, *instantsave_radio, *persistent_radio, *hbox, *spin,
+			*entry_dir, *button, *image, *combo, *help_label;
+		guint i;
+		const GSList *node;
+		gchar *entry_dir_label_text;
+
+		notebook_vbox = gtk_box_new(GTK_ORIENTATION_VERTICAL, 2);
+		inner_vbox = gtk_box_new(GTK_ORIENTATION_VERTICAL, 5);
+		gtk_container_set_border_width(GTK_CONTAINER(inner_vbox), 5);
+		gtk_box_pack_start(GTK_BOX(notebook_vbox), inner_vbox, TRUE, TRUE, 5);
+		gtk_notebook_insert_page(GTK_NOTEBOOK(notebook),
+			notebook_vbox, gtk_label_new(_("Untitled Document Save")), NOTEBOOK_PAGE_UNTITLEDDOC);
+
+		disabled_radio = gtk_radio_button_new_with_mnemonic(NULL, _("Disabled"));
+		pref_widgets.untitled_doc_disabled_radio = disabled_radio;
+		gtk_label_set_mnemonic_widget(GTK_LABEL(label), disabled_radio);
+		gtk_button_set_focus_on_click(GTK_BUTTON(disabled_radio), FALSE);
+		gtk_container_add(GTK_CONTAINER(inner_vbox), disabled_radio);
+		g_signal_connect(disabled_radio, "toggled",
+			G_CALLBACK(radio_toggled_cb), GINT_TO_POINTER(NOTEBOOK_UNTITLEDDOC_RADIO_DISABLED));
+
+		/* Instantsave */
+
+		instantsave_radio = gtk_radio_button_new_with_mnemonic_from_widget(
+			GTK_RADIO_BUTTON(disabled_radio), _("Instant Save"));
```

Comment:
What about "Instant Save After Creation" here so it's clear when this happens (and that saving happens just once compared to persistent documents)?

Reply:
done